test/v1/validation/CountryCodeValidatorSpec.scala

```diff
@@ -26,10 +26,14 @@ class CountryCodeValidatorSpec extends BaseSpec {
 
   "Country Code Validation" when {
 
-    "Country Code is supplied and is not the correct length" in {
+    "Country Code is supplied and is not the correct length as its too short" in {
       val model  = CountryCodeModel("A")
-      leftSideError(model.validate).errorMessage shouldBe
-        errorMessages(CountryCodeLengthError(model).errorMessage, CountryCodeValueError(model).errorMessage)
+      leftSideError(model.validate).errorMessage shouldBe CountryCodeLengthError(model).errorMessage
```

Comment:
So we want to retain the test that both errors get returned, we can do this by using the element parameter of leftSideError:

```
leftSideError(model.validate, 0).errorMessage shouldBe CountryCodeLengthError(model).errorMessage
leftSideError(model.validate, 1).errorMessage shouldBe CountryCodeValueError(model).errorMessage
```